### ISSUE TITLE
startup sound

### ISSUE THREAD

**Issue by combatopera** [combatopera/pym2149#19]
gameboy has one

**Comment by combatopera** [comment]
suggest budgie extra life dosound

**Comment by combatopera** [comment]
perhaps also fix initial DC click - descope for now, it's much less noticeable when accompanied by a sound effect

**Comment by combatopera** [comment]
* implement as an Unpitched that is sent through the midi subsystem on startup
  * will it ever noteoff? - this may only be a cosmetic concern, mediation should be able to handle random notes that stay on forever
  * depends on #38 so that channels.py knows it wants the first 3 channels
  * also depends on #39 as dosound handles its own setup
  * subsequent note on any of the first 3 channels must cancel dosound on all 3 channels
    * not sure how to do this, perhaps #41 will make it more obvious

**Comment by combatopera** [comment]
descope for midi, implement for lc instead as it's much much simpler

**Comment by combatopera** [comment]
implemented for lc. this does mean frameindex is already nonzero when we get real sections, but that's another ticket
